h/buildext.py

```diff
@@ -93,21 +117,14 @@ def chrome_manifest(request):
         version = h.__version__
         version_name = 'Official Build'
 
-    src = request.resource_url(request.context)
-
-    # We need to use only the host and port for the CSP script-src when
```

Comment:
You've removed this behaviour. Unless I'm misunderstanding something that will mean that if you now build a development extension (i.e. one where assets are loaded from localhost) then the CSP checks for the extension will fail when loading assets.

Or is loading assets from the application no longer supported? Or perhaps this was a bug in Chrome that has since been fixed?